cypress/e2e/item/authorization/itemLogin/utils.ts

```diff
@@ -13,5 +13,6 @@ export const addItemLoginSchema = (
     id: v4(),
     createdAt: '2021-08-11T12:56:36.834Z',
     updatedAt: '2021-08-11T12:56:36.834Z',
+    status: 'active',
```

Comment:
```suggestion
    status: ItemLoginSchemaStatus.Active,
```
Or use `ItemLoginSchemaFactory`